app/library/api_schemas.py

```diff
@@ -20,7 +20,10 @@
     "properties": {
         "name": {"type": "string", "pattern": r"^[a-zA-Z0-9 _-]{1,50}$"},
         "friendly_name": {"type": "string", "pattern": r"^[a-zA-Z0-9_-]{1,50}$"},
-        "description": {"type": "string", "pattern": r"^[\?\!\.,a-zA-Z0-9 _-]{,2000}$"},
+        "description": {
```

Comment:
Weird, the two regex are identical, not sure what happened for pre-commit to do this automatically